<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>produitw</class>
 <widget class="QMainWindow" name="produitw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>891</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajout</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>311</width>
        <height>431</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter Client</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Id</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="id">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="prix1">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>170</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="date">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>130</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="type1">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>210</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>pharmacy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>parapharmacy</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="ajout">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>380</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajout</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>QUANTITE</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="quantite">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>250</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>380</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="temp1">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>300</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="humid1">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>340</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>290</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>temp ideale</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>340</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>humid ideale</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>461</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>notif</string>
      </property>
      <widget class="QPushButton" name="notif">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>150</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Notif</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>251</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="Historique">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>380</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Historique</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>afficher</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>821</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="aff">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>430</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>25</y>
        <width>56</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>id</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>prix</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>date</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>30</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>id</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>prix</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>date</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="rech">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>30</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="PDF">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>430</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rech_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>440</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="supprmier">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>430</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>supprmier</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Stats</string>
     </attribute>
     <widget class="QPushButton" name="stats">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Stats</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>621</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="stats_3"/>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>prix</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>type</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="TEMPETHUMID">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>230</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TEMP ET HUMID</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditTemp">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>310</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditHumid">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>390</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>979</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
